css/index.css

```
@@ -0,0 +1,102 @@
+body {
+  font-family: sans-serif;
+  background-color: #FFEEDE;
+  ;
+}
+
+.slide__foodSpin {
+  position: relative;
+  width: 600px;
+  height: 600px;
+  margin: 0 auto;
+  margin-top: 15%;
+}
+
+.carousel__foodSpin {
```

No veo tampoco una clase `.carousel`. Tendrias que cambiar las clases de aqui hasta el final del archivo
```suggestion
.carousel-food-spin {
```